dist.js

```diff
@@ -102,31 +171,42 @@
       if (to) {
         setValue(type, to, false);
         selectElements[type].value = to;
-        unsetBtn.removeAttribute('hidden');
+        showUnsetBtn();
       }
     };
 
     const initMode = () => {
       let to = localStorage.getItem(store.mode);
 
       if (to) {
+        const modeDict = {
+          1: modeLight,
+          [-1]: modeDark
+        };
         setValue('mode', to);
-        unsetBtn.removeAttribute('hidden');
+        showUnsetBtn();
+        modeDict[to].checked = true;
+      } else if (modeAuto) {
+        modeAuto.checked = true;
       }
-    }; // init & events
+    };
+    /* init defaults */
 
 
     document.onload = initMenu();
     document.onload = initMode();
-    modeToggle.addEventListener('click', () => changeMode());
-    unsetBtn.addEventListener('click', () => clearColors());
+    /* attach eventlistners */
+
+    invertBtn && invertBtn.addEventListener('click', () => toggleMode());
```

Comment:
this is clever - i didn't know you could do that

Reply:
I think it's just a more terse way to execute things conditionally rather than using a ternary or more explicit if blocks.